src/nvim/buffer_defs.h

```diff
@@ -1470,4 +1470,10 @@ static inline int win_hl_attr(win_T *wp, int hlf)
 #define CHANGEDTICK(buf) \
   (=== Include buffer.h & use buf_(get|set|inc) _changedtick ===)
 
+// Struct passed to get_v_event() and restore_v_event().
+typedef struct {
+  bool sve_did_save;
+  hashtab_T sve_hashtab;
+} save_v_event_T;
+
```

Comment:
This probably shouldn't be put at the bottom of this file, especially considering the `Macros defined in Vim, but not in Neovim` comment above.

Reply:
Moved it up a bit, I hope the new place is good for you.